Modules/@babylonjs/react-native-iosandroid/android/build.gradle

```diff
@@ -94,7 +94,6 @@ android {
     }
     externalNativeBuild {
         cmake {
-            version '3.19.6'
```

Comment:
Why this change?

Reply:
Ugh, sorry, this isn't supposed to be there either. I run with a newer version of cmake.